src/routes/news/components/List.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+import { FlatList } from 'react-native'
+import styled from 'styled-components/native'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+
+const Loader = styled.ActivityIndicator`
+  margin-top: 7px;
+`
+const keyExtractor = (item, index) => `${index}`
+
+interface ListPropTypes {
+  items: Array<LocalNewsModel | TunewsModel>,
+  renderItem: ({
+    cityCode: string,
+    language: string,
+    item: LocalNewsModel
+  }) => React$Node,
+  isFetchingMore: boolean,
+  getMoreItems: () => void,
+  renderNoItemsComponent: () => React$Node
+}
+
+const List = ({
+  items,
+  renderItem,
+  isFetchingMore,
+  getMoreItems,
+  renderNoItemsComponent
+}: ListPropTypes) => {
+  return (
+    <FlatList
```

Comment:
What is the difference to a scrollview? Do we already use `FlatList` anywhere else?

Reply:
Better rendering performance like doesn't load all data in RAM and better scrolling experience for long lists